### ISSUE TITLE
Some errors should be fixed in this repo

### ISSUE THREAD

**Issue by lyqun** [xuekt98/BBDM#47]
1. pytorch-fid is not in `environment.yml`, but should be installed (pip install pytorch-fid).
2. `./Template-BBDM.yaml`, Line 82, `image_size` should be 256 (be consistent with Line 20).
3. `./Template-LBBDM-f8.yaml`, Line 56, `.ckpt` should be `.pth` (also in `f4` and `f16`).
4. `./model/VQGAN/vqgan.py`, Line 64, `state_dict` should be `model`.

**Comment by yejy53** [comment]
Do you know why if all the pictures that come out are abnormal?
![image](https://github.com/xuekt98/BBDM/assets/83576010/a2a0dae2-cfc6-421c-be6a-82dca7143ed9)

**Comment by yuanshuai-alt** [comment]
> 你知道为什么如果所有出来的图片都是异常的吗？ ![image](https://private-user-images.githubusercontent.com/83576010/317168619-a2a0dae2-cfc6-421c-be6a-82dca7143ed9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI1NzkxODQsIm5iZiI6MTcxMjU3ODg4NCwicGF0aCI6Ii84MzU3NjAxMC8zMTcxNjg2MTktYTJhMGRhZTItY2ZjNi00MjFjLWJlNmEtODJkY2E3MTQzZWQ5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDA4VDEyMjEyNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk1Mjg2NDgyMTY3M2ViNDA3YmY3OTFiZGQyM2E5Y2NmNjk3MGQ0MjI4YWI2NGRhZTE0MTIyZmYzZWVhNDI5NTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.C21uAVD9uZtdIQ8Wt8QEi1HDYsBeBB5DLTCiFBvElEA)

您好请问解决了吗？

**Comment by yejy53** [comment]
No, but it was normal when I tested it, but the training results were all abnormal.

**Comment by szh404** [comment]
> No, but it was normal when I tested it, but the training results were all abnormal.

I also have same problom, and it remained to be same and abnormal when I tested it. Can you tell me how you tested, was there any tricks? Thank you!

**Comment by leoxthomas** [comment]
Hi there,

I'm experiencing the same issue. Images look abnormal in both training and testing. Has anyone been able to fix this?
